### ISSUE TITLE
menu-button: keyboard focus indicator disappears on menu items (Safari)

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1720]
# Bug Report

## Description

On Safari (v15.6), the keyboard focus indicator disappears when navigating menu items with arrow keys.

https://opensource.ebay.com/ebayui-core/?path=/story/buttons-ebay-filter-menu-button--standard&args=type:radio
https://opensource.ebay.com/ebayui-core/?path=/story/buttons-ebay-menu-button--standard

## Workaround

None.

**Comment by saiponnada** [comment]
Waiting for the make-up changes to see if this issue will be fixed automatically. Currently no focus issue on skin & safari.

**Comment by ianmcburnie** [comment]
> Waiting for the make-up changes to see if this issue will be fixed automatically. Currently no focus issue on skin & safari.

This doesn't appear to be a makeup related issue. Makeup menu-button is working fine for me.

**Comment by saiponnada** [comment]
@ianmcburnie, Its working as expected in [storybook sandbox mode](https://opensource.ebay.com/ebayui-core/iframe.html?id=buttons-ebay-menu-button--default&viewMode=story) and also on [skin](https://opensource.ebay.com/skin/#menu-button).  Not really sure why its failing in controls mode. Weird I haven't noticed any styles being overridden.

**Comment by saiponnada** [comment]
As discussed, removing the priority for now as this is working in sandboxed mode.
